kiva/agg/src/graphics_context.i

```diff
@@ -839,15 +840,26 @@ namespace kiva {
                 """
                 from PIL import Image
 
+                FmtsWithDpi = ('jpg', 'png', 'tiff', 'jpeg')
                 FmtsWithoutAlpha = ('jpg', 'bmp', 'eps', "jpeg")
                 size = (self.width(), self.height())
                 fmt = self.format()
 
+                if pil_options is None:
+                    pil_options = {}
+
+                file_ext = filename[-3:].lower() if isinstance(filename, str) else ''
+                if (file_ext in FmtsWithDpi or
+                        (file_format is not None and
+                         file_format.lower() in FmtsWithDpi)):
+                    dpi = int(72 * self.base_scale)
```

Comment:
where exactly does the 72 come from?  could it be wrapped in a variable just so it has a name?